[2024-07-18T16:08:38.919+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-18T16:08:38.956+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_fill_procedure.start_process manual__2024-07-18T16:08:38.218889+00:00 [queued]>
[2024-07-18T16:08:38.971+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_fill_procedure.start_process manual__2024-07-18T16:08:38.218889+00:00 [queued]>
[2024-07-18T16:08:38.973+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-18T16:08:38.992+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): start_process> on 2024-07-18 16:08:38.218889+00:00
[2024-07-18T16:08:38.999+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'run_fill_procedure', 'start_process', 'manual__2024-07-18T16:08:38.218889+00:00', '--job-id', '459', '--raw', '--subdir', 'DAGS_FOLDER/create_and_fill_proc.py', '--cfg-path', '/tmp/tmpde317yl3']
[2024-07-18T16:08:39.002+0000] {standard_task_runner.py:91} INFO - Job 459: Subtask start_process
[2024-07-18T16:08:39.010+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=7251) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-18T16:08:39.012+0000] {standard_task_runner.py:63} INFO - Started process 7252 to run task
[2024-07-18T16:08:39.074+0000] {task_command.py:426} INFO - Running <TaskInstance: run_fill_procedure.start_process manual__2024-07-18T16:08:38.218889+00:00 [running]> on host 7c7e02e57379
[2024-07-18T16:08:39.187+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_fill_procedure' AIRFLOW_CTX_TASK_ID='start_process' AIRFLOW_CTX_EXECUTION_DATE='2024-07-18T16:08:38.218889+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-18T16:08:38.218889+00:00'
[2024-07-18T16:08:39.190+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-18T16:08:39.215+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-07-18T16:08:44.246+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-18T16:08:44.253+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-18T16:08:44.269+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=run_fill_procedure, task_id=start_process, run_id=manual__2024-07-18T16:08:38.218889+00:00, execution_date=20240718T160838, start_date=20240718T160838, end_date=20240718T160844
[2024-07-18T16:08:44.299+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-18T16:08:44.311+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-07-18T16:08:44.325+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-18T16:08:44.328+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
